src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -50,9 +50,17 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 return true;
             }
         }
-
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        
+        
+        if (!player.isOp())
++        {

```

Comment:
It looks like its been coppied from another commit, because Github adds them in...